src/components/ScreenStackItem.tsx

```diff
@@ -86,17 +101,16 @@ function ScreenStackItem({
       activityState={activityState}
       stackPresentation={stackPresentation}
       hasLargeHeader={headerConfig?.largeTitle ?? false}
-      {...rest}
-    >
+      style={[style, internalScreenStyle]}
+      {...rest}>
       {isHeaderInModal ? (
         <ScreenStack style={styles.container}>
           <Screen
             enabled
             isNativeStack
             activityState={activityState}
             hasLargeHeader={headerConfig?.largeTitle ?? false}
-            style={StyleSheet.absoluteFill}
-          >
+            style={[StyleSheet.absoluteFill, internalScreenStyle]}>
```

Comment:
Do we need this style here?

Reply:
I think you're right. I haven't added it there initially (in the code that introduced the formsheets), because this Screen here is not the bottom sheet (it is not responsible for carrying out the presentation)!

I think this is good catch, but I'm only 95% sure. @maciekstosio can we test this?

Reply:
Yeah, I played with it and it seems that's not necessary. Removed in: https://github.com/software-mansion/react-native-screens/pull/2419/commits/40b78a8ad30ce56e74202ecead2248f7f92a5d34